perceval/rendering/pdisplay.py

```diff
@@ -284,13 +284,17 @@ def _get_sub_figure(ax: Axes3D, array: numpy.array, basis_name: list):
     ax.view_init(elev=30, azim=45)
 
 
-def pdisplay_tomography_chi(qpt: ProcessTomography, output_format: Format = Format.MPLOT, precision=1E-6):
-    if output_format != Format.MPLOT:
-        raise TypeError("Tomography plot only support MPLOT")
+def pdisplay_tomography_chi(qpt: ProcessTomography, output_format: Format = Format.MPLOT, precision=1E-6,
+                            render_size=None):
```

Comment:
Can we add the parameter `render_size` in the pdisplay method docstring pretty please ?